ultimatejobweb/models.py

```diff
@@ -13,5 +13,14 @@ class Job(models.Model):
     job_title = models.CharField(max_length=128)
     description_url = models.URLField()
 
+    @classmethod
+    def push_data_api(cls, company_name_api, title_list, url_list):
+        combined = zip(title_list, url_list)
+        combined_list = list(combined)
+        title_list, url_list = zip(*combined_list)
+
+        for job_title, description_url in combined_list:
+            company_job = Company.objects.get(company_name=company_name_api)
```

Comment:
@BarBenNaim 
I think this method should be part of class company
And than you could use the cls member

  for job_title, description_url in combined_list:
            company_job = cls.objects.get(company_name=company_name_api)

@BarBenNaim do you see it now?

Reply:
@Yarboa Yes Im try

Reply:
`    @classmethod
    def push_data_api(cls, company_name_api, title_list, url_list):
        combined = zip(title_list, url_list)
        combined_list = list(combined)
        title_list, url_list = zip(*combined_list)

        for job_title, description_url in combined_list:
            company_job = Company.objects.get(company_name=company_name_api)
            cls(company=company_job, job_title=job_title, description_url=description_url).save()`

Reply:
Ack